tests/fast-integration/utils/index.js

```diff
@@ -578,6 +578,40 @@ function debug() {
   }
 }
 
+function toBase64(s) {
+  return Buffer
+    .from(s)
+    .toString("base64");
+}
+
+function genRandom(len) {
+  let res = "";
+  const chrs = "abcdefghijklmnopqrstuvwxyz0123456789";
+  for (let i = 0; i < len; i++ ) {
+    res += chrs.charAt(Math.floor(Math.random() * chrs.length));
+  }
+
+  return res;
+}
+
+function shouldRunSuite(suite) {
+  debug("COMPASS_INTEGRATION_ENABLED", process.env.COMPASS_INTEGRATION_ENABLED);
+  switch(suite) {
+    case "commerce-mock-compass":
```

Comment:
Here you have `commerce-mock-compass`, in `tests/fast-integration/test/2-commerce-mock.js` you have `commerce-mock` - it's a bug or feature? :D

Reply:
Yes, this will make sense when I merge the compass suite, so it is a feature ;)